fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/ApiRequestImpl.java

```diff
@@ -680,12 +600,11 @@ public ResponseFormatType getFormat() {
     }
 
     @Override
-    public long getAsyncAfter() {
+    public Long getAsyncAfter() {
         return asyncAfter;
     }
 
-    @Override
-    public PaginationParameters getDefaultPagination() {
+    public static PaginationParameters getDefaultPagination() {
```

Comment:
This is dangerous. We're not using `getDefaultPagination` anywhere, let alone overriding, so I don't care that much, but we'd probably be better off deprecating this method, adding a static version, and using the static version, rather than just making this method static.

Reply:
the default pagination is particular to the use case.
In this case the ApiRequests have a configuration driven Default Pagination.

Reply:
Moving all calls to the method to references to the static instance variable.  There are configurable default paginations and they should probably be centralized.  But not by making this static.  Definitely not by making this static.